canonical/ubuntu.com
# WD-7125 - add loading spinner to static forms

## Done

Main issue addressed:
- Make loading spinners work on static forms

Fly-by work:
- Make static-forms globally available
- Refactor static forms structure and fix some bugs
- Remove static-forms.js being included on individual document level
- Fixes some incorrect ids/names on a static form

## QA

- this form has everything you need to test, but also test on some others https://ubuntu-com-13466.demos.haus//kernel/real-time/contact-us
- Check that static-forms.js loads on all static forms (it is available globally but will only run the script if there is a static form)
- Check that all the functionality still works: 'other Inputs', 'commentFromLead', 'setting primary language' and now 'loading spinners'
- Check out the random form I updated, and see I removed duplicate use of name/ids to make it more clear for marketing

## Issue / Card

[WD-7125](https://warthogs.atlassian.net/browse/WD-7125)


[WD-7125]: https://warthogs.atlassian.net/browse/WD-7125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
